v1_14_4_R1/src/main/java/com/lishid/orebfuscator/nms/v1_14_2_R1/NmsManager.java

```diff
@@ -0,0 +1,617 @@
+/**
+ * @author lishid
+ * @author Aleksey Terzi
+ *
+ */
+
+package com.lishid.orebfuscator.nms.v1_14_2_R1;
+
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.World;
+import org.bukkit.craftbukkit.v1_14_R1.CraftWorld;
+import org.bukkit.craftbukkit.v1_14_R1.block.data.CraftBlockData;
+import org.bukkit.craftbukkit.v1_14_R1.entity.CraftPlayer;
+import org.bukkit.craftbukkit.v1_14_R1.util.CraftChatMessage;
+import org.bukkit.entity.Player;
+
+import com.lishid.orebfuscator.nms.IBlockInfo;
+import com.lishid.orebfuscator.nms.IChunkCache;
+import com.lishid.orebfuscator.nms.INBT;
+import com.lishid.orebfuscator.nms.INmsManager;
+import com.lishid.orebfuscator.types.BlockCoord;
+import com.lishid.orebfuscator.types.ConfigDefaults;
+
+import net.minecraft.server.v1_14_R1.Block;
+import net.minecraft.server.v1_14_R1.BlockPosition;
+import net.minecraft.server.v1_14_R1.Chunk;
+import net.minecraft.server.v1_14_R1.ChunkProviderServer;
+import net.minecraft.server.v1_14_R1.IBlockData;
+import net.minecraft.server.v1_14_R1.IChatBaseComponent;
+import net.minecraft.server.v1_14_R1.Packet;
+import net.minecraft.server.v1_14_R1.TileEntity;
+import net.minecraft.server.v1_14_R1.WorldServer;
+
+public class NmsManager implements INmsManager {
+	private static final int BITS_PER_BLOCK = 14;
```

Comment:
Not 100% sure but indicators I use, indicate that this bits per block is still 14, so good news there